docs/vision/utility/distribution.md

```diff
@@ -0,0 +1,33 @@
+# Distribution
+
+DropIt provides multiple ways to distribution custom tokens to the Polkadot community.
+
+## Polkadot Snapshot
+
+Using the state root of Polkadot, token distributors are able to distribute tokens to existing DOT holders exactly in proportion to the amount of DOT they are holding on the Polkadot relay chain.
```

Comment:
As Polkadot also have the parachain headers in its state, it can actually also proof state in the parachains.